### PR TITLE
Entswm491

### PR DESCRIPTION
**Pull request by penehyba**
Test is based on ScalingIT, three pods are rolled thanks to redeploying deployment config.

### CODE REVIEW

File: common/src/main/java/io/thorntail/openshift/ts/common/arquillian/OpenShiftUtil.java [hunk 1]
```diff
@@ -22,10 +22,14 @@
         this.oc = oc;
     }
 
-    private List<Pod> listPodsForDeploymentConfig(String deploymentConfigName) {
+    public List<Pod> listPodsForDeploymentConfig(String deploymentConfigName) {
+        return listPodsForLabel("deploymentconfig",deploymentConfigName);
+    }
+
+    public List<Pod> listPodsForLabel(String labelName, String labelValue) {
```
Comment (Ladicek) [inline common/src/main/java/io/thorntail/openshift/ts/common/arquillian/OpenShiftUtil.java]: Why this change?

File: pom.xml [hunk 2]
```diff
@@ -34,7 +35,7 @@
         <version.io.fabric8.fabric8-maven-plugin>4.1.0</version.io.fabric8.fabric8-maven-plugin>
         <version.io.fabric8.openshift-client>4.1.1</version.io.fabric8.openshift-client>
         <version.io.rest-assured>3.1.1</version.io.rest-assured>
-        <version.io.thorntail>2.4.1.Final-SNAPSHOT</version.io.thorntail>
+        <version.io.thorntail>2.5.1.Final-SNAPSHOT</version.io.thorntail>
```
Comment (Ladicek) [inline pom.xml]: 2.5.0.Final should work?

File: scaling/src/test/java/io/thorntail/openshift/ts/scaling/ScalingIT.java [hunk 3]
```diff
@@ -77,9 +74,9 @@ public void scaleDownToZero() {
         for (int i = 0; i < 100; i++) {
             given()
                     .baseUri(url)
-            .when()
+                    .when()
                     .get()
-            .then()
+                    .then()
```
Comment (Ladicek) [inline scaling/src/test/java/io/thorntail/openshift/ts/scaling/ScalingIT.java]: Why this change?

**Comment by Ladicek** [comment]
Test looks good! I'll try it locally before merging.

File: common/src/main/java/io/thorntail/openshift/ts/common/arquillian/OpenShiftUtil.java (hunk 1 above)
Comment (penehyba) [reply common/src/main/java/io/thorntail/openshift/ts/common/arquillian/OpenShiftUtil.java]: This change in OpenShiftUtil was (and probably will be) useful..but after refactoring - this is no longer used, reverted.
modified to 2.5.0.Final
spaces aligned by IDEA reverted.

**Comment by penehyba** [comment]
I tried it even on Thorntail 2.x OpenShift TS using OpenJDK 8 and result looks good http://pastebin.test.redhat.com/796901

**Review by Ladicek — APPROVED** [review]

**Comment by Ladicek** [comment]
Thanks for the patience :-)
